content/pt-br/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Implantar e acessar o Kubernetes Dashboard
+description: >-
+  Implementa a interface web do usuário (Kubernetes Dashboard) e acessá-lo.
+content_type: concept
+weight: 10
+card:
+  name: tasks
+  weight: 30
+  title: Usar a interface de usuário do Dashboard web
+---
+
+
+<!-- overview -->
+
+Dashboard é uma interface de usuário Kubernetes baseada na Web.
```

Comment:
I think this can be: 'Dashboard é uma interface de usuário web para o Kubernetes'  what you think?